extensions/amp-autocomplete/amp-autocomplete.md

```diff
@@ -119,8 +119,8 @@ Example:
     </td>
   </tr>
   <tr>
-    <td width="40%"><strong>highlight-user-entry (optional)</strong></td>
```

Comment:
`highlight-user-entry` is still allowed in validation. Is it intended to remove this from documentation?

Reply:
Not at all, thanks for catching this!